Assets/Source/Entities/Behaviors/Influential.cs

```diff
@@ -1,14 +1,33 @@
 using UnityEngine;
 using System.Collections;
+using System.Collections.Generic;
 
 public class Influential : MonoBehaviour
 {
+	private static HashSet<Influential> queens = new HashSet<Influential>();
+	public static HashSet<Influential> Queens {
```

Comment:
Also I think this will give you the same thing. I think...?
```csharp
public static HashSet<Influential> Queens { get; private set;}
...
public Influential()
{
	Queens = new HashSet<Influential>();
}
```